src/_scss/pages/analystGuide/analystGuide.scss

```diff
@@ -18,46 +18,103 @@
-      margin: 0 0.625rem;
-    }
+  .analyst-guide__share-dl-wrapper {
+    display: flex;
+    vertical-align: center;
+    .analyst-guide__share-wrapper {
+      &:hover {
+        color: $color-primary-darker;
+      }
+      color: $color-primary;
+      margin-top: $global-margin * 4;
+      margin-left: -24px;
+      margin-right: -12px;
+      @media (min-width: $medium-screen) {
+        margin-left: -40px;
+        margin-right: -12px;
+      }
 
-    // these affect the share dropdown position
-    // overriding the styles in data-transparency-ui Picker, sent by ShareIcon
-    .usa-dt-picker .usa-dt-picker__dropdown-container ul {
-      @media (min-width: $small-screen) and (max-width: 374px) {
-        right: 110px !important;
+      .usda-share-icon .usa-dt-picker__dropdown-container {
+        @include button-link;
       }
-      @media (min-width: 375px) and (max-width: 474px) {
-        right: 132px !important;
+  
+      .usa-dt-picker .usa-dt-picker__dropdown-container .usa-dt-picker__button {
+        margin: 0 0.5rem;
       }
-      @media (min-width: 475px) and (max-width: 574px) {
-        right: 232px !important;
+
+      .usda-share-icon .usa-dt-picker__dropdown-container .usa-dt-picker__button svg {
+        &:hover {
+          color: $color-primary-darker !important;
+        }
       }
-      @media (min-width: 575px) and (max-width: 674px) {
-        right: 332px !important;
+  
+      // these affect the share dropdown position
+      // overriding the styles in data-transparency-ui Picker, sent by ShareIcon
+      .usa-dt-picker .usa-dt-picker__dropdown-container ul {
+        @media (min-width: $small-screen) and (max-width: 374px) {
+          right: 110px !important;
+        }
+        @media (min-width: 375px) and (max-width: 474px) {
+          right: 132px !important;
+        }
+        @media (min-width: 475px) and (max-width: 574px) {
+          right: 232px !important;
+        }
+        @media (min-width: 575px) and (max-width: 674px) {
+          right: 332px !important;
+        }
+        @media (min-width: 675px) and (max-width: 767px) {
+          right: 444px !important;
+        }
+        @media (min-width: 768px) and (max-width: 868px) {
+          right: 525px !important;
+        }
+        @media (min-width: 869px) and (max-width: $medium-screen) {
+          right: 625px !important;
+        }
       }
-      @media (min-width: 675px) and (max-width: 767px) {
-        right: 444px !important;
+    }
+  
+    .analyst-guide__download-wrapper {
+      display:inline-block;
+      color: $color-primary;
+      background-color: white;
+      margin-top: $global-margin * 4;
+      left: 10px;
+      &:hover {
+        color: $color-primary-darker;
       }
-      @media (min-width: 768px) and (max-width: 868px) {
-        right: 525px !important;
+      @media (min-width: $medium-screen) {
+        margin-left: 0px;
```

Comment:
The 'px' unit isn't necessary here since the value is zero.  Same for the class below.